### PR TITLE
Fix Python 3 setup on non-UTF-8 systems

### PR DESCRIPTION
**Pull request by fkrull**
`setup.py` was opening `webencodings/__init__.py` using the system 8-bit encoding, so not UTF-8 on Windows. This change is compatible with Python 2.6 and 2.7.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
 from setuptools import setup, find_packages
+from io import open
```
Comment (gsnedders) [inline setup.py]: I'd rather we didn't overwrite the `open` built-in, because I just find that confusing.

**Comment by gsnedders** [comment]
Do we not have the same issue with `LONG_DESCRIPTION`, too? (really less of a problem, though, except for PyPI)

**Comment by fkrull** [comment]
> Do we not have the same issue with LONG_DESCRIPTION, too?

Ah, you're right. I missed that one. README.rst was just ASCII though so it didn't throw.

**Comment by gsnedders** [comment]
https://github.com/gsnedders/python-webencodings/commit/835589f60010a8365f8875e7f195d1ca1e02e8ad

**Pull request closed without merging**